include/solvers/gls_navier_stokes.h

```diff
@@ -142,13 +142,13 @@ class GLSNavierStokesSolver
    * Set-up AMG preconditioner
    */
   void
-  setup_AMG();
+  setup_AMG(int temp_precond_fill_amg);
```

Comment:
```suggestion
  setup_AMG(int amg_ilu_preconditioner_fill);
```
Don't hesitate to use variable name that have a more extensive meaning. Additionnally, the fill level here is not temporary, it's the fill level used for the pre-conditioner itself.

Reply:
Additionnaly. Should be a const argument if it does not change.